src/libraries/System.Formats.Tar/tests/TarWriter/TarWriter.Tests.cs

```diff
@@ -350,5 +351,20 @@ private int GetChecksumForFormatSpecificFields(TarEntry entry, TarEntryFormat fo
             // Gnu BlockDevice: 623 + 1004 + 686 + 1142 = 3455
             return checksum;
         }
+
+        [Fact]
+        void Verify_Size_RegularFile_Empty()
```

Comment:
nit: the test methods should be `public`

Reply:
Yup, for consistency and repo conventions. However, it is known to work with xunit: https://github.com/xunit/xunit/discussions/2438#discussioncomment-3218074 :)